web/src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -156,8 +208,9 @@ export default {
         return;
       }
 
-      // Create the download link in itemDownloads for each item in items
+      // Create download and delete links in local data for each item in items
       this.itemDownloads = {};
+      this.itemDeletes = {};
```

Comment:
I think these might make more sense joined into one map that is accessed for both downloading and deleting. I don't feel strongly, just a thought.

Reply:
See cf081408de80cb931771ef3c7279966d62c5d763.

I am not sure this is really better than what we had (it is not terrible either), but let me know what you think.

Reply:
I assume this thread is resolved; open it back up if not.